presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -417,11 +417,7 @@ static boolean isHudiParquetInputFormat(InputFormat<?, ?> inputFormat)
 
     private static boolean shouldUseFileSplitsForHudi(InputFormat<?, ?> inputFormat, HoodieTableMetaClient metaClient)
     {
-        if (inputFormat instanceof HoodieParquetRealtimeInputFormat) {
-            return true;
-        }
-
-        return metaClient.getTableConfig().getBootstrapBasePath().isPresent();
+        return inputFormat instanceof HoodieParquetRealtimeInputFormat;
```

Comment:
I get that we want this to be the default. but should n't there be a path where users want to query _hoodie_record_key and other fields - and we need to stitch them together?